### DISCUSSION TITLE
Error in development yarn build

### DISCUSSION THREAD

**Discussion by markns** [getredash/redash#6220] [General]
Hi, I'm trying to spin up a development env, as I'd like to try making a PR to add query_tags to the Snowflake connector, but I'm running into an issue when I run `yarn --frozen-lockfile` according to these instructions: https://redash.io/help/open-source/dev-guide/docker

```
error An unexpected error occurred: "ENOTDIR: not a directory, lstat '/Users/markns/workspace/getredash/redash/node_modules/react-ace/dist/react-ace.min.js/main.js'".
info If you think this is a bug, please open a bug report with the information provided in "/Users/markns/workspace/getredash/redash/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

This file exists: 
```/Users/markns/workspace/getredash/redash/node_modules/react-ace/dist/react-ace.min.js```

but not this: ```/Users/markns/workspace/getredash/redash/node_modules/react-ace/dist/react-ace.min.js/main.js```

```
❯ node --version
v16.20.1

❯ yarn --version
1.22.19
```

I'm not very familiar with the React side of things. Any ideas?

**Comment by justinclift** [comment]
Ahhh yeah.  The developer setup docs on the website are pretty busted (they'll get fixed at some point).

In the meantime, we're all using these docs from the wiki, which are known to work properly:

https://github.com/getredash/redash/wiki/Local-development-setup

**Comment by justinclift** [comment]
Hmmm, from the `/Users/markns/ ...` text fragment, it looks like you're using macOS.

@gaecoli also runs macOS, so *might* be useful for pointers if you hit issues that seem macOS-specific. :smile:

**Reply by gaecoli** [reply]
Ok.

**Reply by gaecoli** [reply]
Merged!

**Comment by markns** [comment]
Thanks @justinclift and @gaecoli . Not sure if it was the merged fix or the wiki instructions, but I've been able to build 🙏

**Reply by justinclift** [reply]
Note that we're re-creating the merged fix, as there turned out to be a strategic problem with it.

So, you *might* need to re `git pull` from the `master` branch before submitting a PR, as I had to force push our `master` branch without the merged fix.  (super, super rare that happens)
